regolith/helpers/l_membershelper.py

```diff
@@ -67,15 +67,28 @@ def sout(self):
         bad_stati = ["finished", "cancelled", "paused", "back_burner"]
         people = []
         for person in self.gtx["people"]:
-            if rc.current and not person.get('active'):
-                continue
-            people.append(person)
-        #people.sort()
+            if rc.current:
+                if not person.get('active'):
+                    continue
+                people.append(person)
+            elif rc.former:
+                if person.get('active'):
+                    continue
+                people.append(person)
+            else:
+                people.append(person)
+
         for i in people:
             if rc.verbose:
                 print("{}, {} | group_id: {}".format(i.get('name'), i.get('position'), i.get('_id')))
                 print("    orcid: {} | github_id: {}".format(i.get('orcid_id'), i.get('github_id')))
+                for employment in i.get('employment'):
+                    if is_current(employment):
+                        print("    current organization: {}".format(employment.get('organization')))
```

Comment:
This would print better if you dereference organization from institutions:
```
inst = fuzzy_retrieval(gtx["institutions"],["aka", "name", "_id"], position.get("organization")).get("name")
print("    current organization: {}".format(inst))
```

Reply:
@sbillinge I tried the fuzzy retrieval approach but run into an issue where the code did not find gtx['institution'], could you point me in the direction either where I need to create or pull this data from?

Reply:
@sbillinge figured this out and fixed it!

Reply:
good job!.  I see it should have been "institutions", was that the issue?

Reply:
@sbillinge We had to add:
    def sout(self):
        gtx = self.gtx (this one)
That's why it was not recognizing gtx